### PR TITLE
Hotkeys / Keyboard shortcuts implemented

### PR DESCRIPTION
**Pull request by sei-noconnor**

### CODE REVIEW

File: src/app/shared/directives/crucible-hotkey.directive.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    private el: ElementRef
+  ) {}
+  private subscription: Subscription;
+  @Input() hotkeyAction: string;
+  @Output() hotkeyFn: EventEmitter<KeyboardEvent> = new EventEmitter<
+    KeyboardEvent
+  >();
+
+  ngOnChanges(changes: SimpleChanges): void {
+    const test = 0;
```
Comment (sei-ebram) [inline src/app/shared/directives/crucible-hotkey.directive.ts:31]: LGTM flagged this unused

File: src/app/shared/directives/crucible-hotkey.directive.spec.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+describe('CrucibleHotkeyDirectiveDirective', () => {
```
Comment (sei-ebram) [inline src/app/shared/directives/crucible-hotkey.directive.spec.ts:1]: header

File: src/app/shared/directives/crucible-hotkey.directive.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import {
```
Comment (sei-ebram) [inline src/app/shared/directives/crucible-hotkey.directive.ts:1]: header

File: src/app/shared/shared.module.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { CommonModule } from '@angular/common';
```
Comment (sei-ebram) [inline src/app/shared/shared.module.ts:1]: header

**Review by sei-tspencer — APPROVED** [review]

**Review by sei-aschlackman — APPROVED** [review]

**Pull request merged**